packages/checkout-storefront/src/sections/PaymentSection/AdyenDropIn/errorMessages.ts

```diff
@@ -0,0 +1,215 @@
+import { defineMessages } from "react-intl";
+
+export const adyenErrorMessages = defineMessages({
+  refused: {
+    defaultMessage: "The transaction was refused.",
+    id: "AdyenDropIn/errorMessages/VFX2HQ",
+    description: "The transaction was refused.",
+  },
+  acquirerError: {
+    defaultMessage:
+      "The transaction did not go through due to an error that occurred on the acquirer's end.",
+    id: "AdyenDropIn/errorMessages/71wcNF",
+    description:
+      "The transaction did not go through due to an error that occurred on the acquirer's end.",
+  },
+  blockedCard: {
+    defaultMessage: "The card used for the transaction is blocked, therefore unusable.",
+    id: "AdyenDropIn/errorMessages/0NXVfz",
+    description: "The card used for the transaction is blocked, therefore unusable.",
+  },
+  expiredCard: {
+    defaultMessage: "The card used for the transaction has expired. Therefore it is unusable.",
+    id: "AdyenDropIn/errorMessages/qeGblU",
+    description: "The card used for the transaction has expired. Therefore it is unusable.",
+  },
+  invalidAmount: {
+    defaultMessage: "An amount mismatch occurred during the transaction process.",
+    id: "AdyenDropIn/errorMessages/OVcUz1",
+    description: "An amount mismatch occurred during the transaction process.",
+  },
+  invalidCardNumber: {
+    defaultMessage: "The specified card number is incorrect or invalid.",
+    id: "AdyenDropIn/errorMessages/HmkjpZ",
+    description: "The specified card number is incorrect or invalid.",
+  },
+  issuerUnavailable: {
+    defaultMessage:
+      "It is not possible to contact the shopper's bank to authorise the transaction.",
+    id: "AdyenDropIn/errorMessages/H3UOYb",
+    description: "It is not possible to contact the shopper's bank to authorise the transaction.",
+  },
+  notSupported: {
+    defaultMessage:
+      "The shopper's bank does not support or does not allow this type of transaction.",
+    id: "AdyenDropIn/errorMessages/7L60IY",
+    description: "The shopper's bank does not support or does not allow this type of transaction.",
+  },
+  "3DNotAuthenticated": {
+    defaultMessage:
+      "3D Secure authentication was not executed, or it did not execute successfully.",
+    id: "AdyenDropIn/errorMessages/r/nPYl",
+    description: "3D Secure authentication was not executed, or it did not execute successfully.",
+  },
+  notEnoughBalance: {
+    defaultMessage: "The card does not have enough money to cover the payable amount.",
+    id: "AdyenDropIn/errorMessages/dFVdlJ",
+    description: "The card does not have enough money to cover the payable amount.",
+  },
+  acquirerFraud: {
+    defaultMessage: "Possible fraud.",
```

Comment:
```suggestion
    defaultMessage: "The transaction was marked as fraudulent. Please contact the store's customer service to resolve this issue.",
```
If there's the transaction was rejected due to possible fraud I think it's better to redirect customers to shop's customer service, because it can be a false-positive.